merge-headers/index.js

```diff
@@ -10,6 +10,10 @@
  * @return {Headers}
```

Comment:
Totally unrelated, but line 9 has a typo 😅

Reply:
What, headerses? If Headers is the constructor, what's the plural?

Reply:
I guess you have a valid point 🤷‍♂️